ubuntu/content.md

```diff
@@ -10,7 +10,7 @@ Development of Ubuntu is led by Canonical Ltd. Canonical generates revenue throu
 
 # What's in this image?
 
-This image is built from official rootfs tarballs provided by Canonical (specifically, https://partner-images.canonical.com/core/).
+This image is built from official rootfs tarballs provided by Canonical (specifically, https://partner-images.canonical.com/oci/ and https://partner-images.canonical.com/core/).
```

Comment:
`/oci` for >= bionic and `/core` for < bionic could be mentioned, but this is already an improvement. Thanks!

Reply:
Sure, updated :smile: